doc/.vale.ini

```diff
@@ -20,7 +20,7 @@ WordTemplate = \b(?:%s)\b
 Packages = Google
 
 # Define the Ansys vocabulary
-Vocab = ANSYS, PyAnsys
+Vocab = ANSYS
```

Comment:
So, Vale has not been running for some undefined amount of time on our repo because this vocabulary directory does not exist in the repo.  Most disturbing is the doc style check does not report an error in the CI/CD pipeline if Vail dies because of this.  @RobPasMue @jorgepiloto

Reply:
Agreed, this is "vale"'s behaviour but it is completely disturbing. We saw silent core dump errors in the past as well...